### PR TITLE
Update azure-pipelines.yml for Azure Pipelines

### PR DESCRIPTION
**Pull request by Intern426**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -34,3 +34,8 @@ steps:
     jdkArchitectureOption: 'x64'
     publishJUnitResults: true
     testResultsFiles: '**/surefire-reports/TEST-*.xml'
+  env:
+    AZURE_CLIENT_ID: '$(CLIENT_ID)'
+    AZURE_CLIENT_SECRET: '$(CLIENT_SECRET)'
+    AZURE_TENANT_ID: '$(TENANT_ID)'
+    KEY_VALUE: '$(VAULT_URL)'
```
Comment (conniey) [inline azure-pipelines.yml]: What is Key value?

Comment (conniey) [reply azure-pipelines.yml]: Oh. A better name is KEY_VAULT_URL.

Comment (Intern426) [reply azure-pipelines.yml]: Key value is just the Key Vault URL. I need it in another pull request, but for some reason, the build on dev.com will always show that the check working, but when I try to pull it, the check always says it fails. So I decided to separate the two tasks and it seems to like it better

Comment (conniey) [reply azure-pipelines.yml]: Sounds good.

**Pull request merged**